queries/unison/highlights.scm

```diff
@@ -0,0 +1,75 @@
+;; Primitives
+(comment) @comment
+(nat) @number
+(unit) @constant.builtin
+(literal_char) @character
+(literal_text) @string
+(literal_boolean) @boolean
+
+;; Keywords
+[
+  (kw_forall)
+  (unique_kw)
+  (type_kw)
+  (kw_equals)
+  (do)
+] @keyword
+
+(kw_let) @keyword.function
```

Comment:
does let declare *only* functions or a variable/other stuff?

Reply:
let opens a "block" wherever an expression is expected https://www.unison-lang.org/learn/language-reference/blocks-and-statements/

(though you assign variables/functions inside blocks, but that muddies the water a bit because I think *everything* is a function in unison).

Not sure what the best fit is though for capture.

Reply:
Going to move it into the @keyword block instead of keyword.function if that makes sense

Reply:
yeah that's good